app/assets/javascripts/admin/addon/components/site-settings/bool.gjs

```diff
@@ -0,0 +1,39 @@
+import Component from "@glimmer/component";
+import { on } from "@ember/modifier";
+import { action, computed } from "@ember/object";
+import { htmlSafe } from "@ember/template";
+import { isEmpty } from "@ember/utils";
+
+export default class Bool extends Component {
+  @computed("value")
```

Comment:
do we need this computed here?

Reply:
Linking this answer from david on another PR: https://github.com/discourse/discourse/pull/30529/files#r1900690995

Reply:
I was actually able to remove this in another way by using @tracked to get the initial value